examples/02-Merlin-Models-and-NVTabular-applying-to-your-own-dataset.ipynb

```diff
@@ -0,0 +1,688 @@
+{
```

Comment:
<p>Besides filtering categorical and continuous features, we could maybe give more examples about how the schema is used in building the architecture. Such as: </p><p> </p><ul><li>Using the cardinality of categorical features to infer the right <a href="https://github.com/NVIDIA-Merlin/models/blob/a75e81223b2f2a8c5ab7238c7cf27ac77815f7b6/merlin/models/tf/features/embedding.py#L122" target="_blank">embedding_size</a> : </li><li>Infer the<a href="https://github.com/NVIDIA-Merlin/models/blob/29d91e449a1bea5bba9273ee655fe39e53529127/merlin/models/tf/utils/tf_utils.py#L35" target="_blank"> output dimension</a> of input block based on the schema</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>